lib/balanced_vrp_clustering.rb

```diff
@@ -134,15 +187,96 @@ def build(data_set, cut_symbol, cut_ratio = 1.0, options = {})
         self
       end
 
+      def move_limit_violating_dataitems
+        @needed_moving = @items_with_limit_violation.size
+        mean_distance_diff = @items_with_limit_violation.collect{ |d| d[1] }.mean
+        mean_ratio = @items_with_limit_violation.collect{ |d| d[2] }.mean
+
+        @moved_down = 0
```

Comment:
Both `moved_up` and `moved_down` were first implemented like that because they were used elsewhere but apparently, not anymore after the changes. I will correct it accordingly.